python3/tests/test_functional.py

```diff
@@ -1,7 +1,9 @@
 # coding=utf-8
 
 import unittest
-import sys, logging, socket, os
+import sys
+import logging
+import os
```

Comment:
The static analysis tool enforces one import per line

Reply:
That's fine 👍🏽